ufo/templates/layout.html

```diff
@@ -2,9 +2,7 @@
 <html lang="en">
 <head>
   <title>{% block title %}{% endblock %}</title>
-  <link rel="import" href="{{ url_for('static', filename='bower_components/polymer/polymer.html') }}" />
-  <link rel="import" href="{{ url_for('static', filename='bower_components/paper-drawer-panel/paper-drawer-panel.html') }}" />
-  <link rel="import" href="{{ url_for('static', filename='sidebar.html') }}" />
+  <link rel="import" href="{{ url_for('static', filename='vulcanized.html') }}" />
```

Comment:
Yes, it contains every custom element defined in static (sidebar.html, textarea.html, and any future custom ones we defined, such as toggleInput or collapseButton from my other PR's) and every bower_component currently downloaded. This will obviously be way more than any single page needs individually, but the tradeoff is that this file is cached after initial load which means after initial load, there is no need to redownload the vulcanized file. Navigating from the user handler to the proxy server handler for instance will then just pull down the requested template (if modified) and surface the cached vulcanized file.

For the hard data, vulcanized.html is (currently) 731.3 kB and downloading an updated version (I added a space and refreshed the page) took anywhere between 14 and 30 ms on my machine. When it is cached, the size of the total request to re-surface the cached version is only 415 B which takes anywhere from 8 ms to 27 ms on my machine. If I jump back on master without any of these changes though, loading any one element is roughly 412 B from cache and takes somewhere between 3 and 40 ms. There are roughly 30~40 elements per page though, as one element may depend on many more recursively.